### ISSUE TITLE
CPA loss

### ISSUE THREAD

**Issue by Xqq2620xx** [fuying-wang/MGCA#29]
您好！请问一下文章公式8的计算中是不是少了负号？
<img width="510" alt="image" src="https://github.com/HKU-MedAI/MGCA/assets/62050001/4cd46f3a-4b4b-477c-85a1-ce0351caa60f">

代码中好像是有负号的～

谢谢！

**Comment by fuying-wang** [comment]
你好，

谢谢指出，cross entropy应该是有负号的

**Comment by Xqq2620xx** [comment]
谢谢回复。
此外，想确认一下，文中写的所用到的预训练数据pair大概是217k
<img width="901" alt="image" src="https://github.com/HKU-MedAI/MGCA/assets/62050001/156a9560-0178-407a-a07c-decfb18655fc">

但是我运行代码的时候，pretrain_dataset.py中len(path2sent)的大小为232282，请问您在代码运行过程中次数的输出就是217k左右吗？
![da91a14444b21c70237cdc5194c0128a](https://github.com/HKU-MedAI/MGCA/assets/62050001/fa8da726-9361-4adb-be30-f92663245a34)

我用的原数据集，代码也没有修改，想知道是否遗漏了什么步骤导致数量没有对齐。

谢谢！
